en/docs/tutorials/scenarios/scenario11-graphql.md

```diff
@@ -1,4 +1,42 @@
 # Scenario 11 - GraphQL Support
 
-!!! Note
-    Content to be added.
\ No newline at end of file
+## User Story
+
+**Quantis is more focused on providing more capability to the developer community. They expect the developer community to build their own mobile applications and web apps to use their APIs. To make this process much easier, Quantis wants to expose GraphQL API to the public.**
```

Comment:
Let's unbold all of these. Readability isn't great with full sections bold and bold is better used when just highlighting something in a sentence.

Reply:
removed in all the docs